### DISCUSSION TITLE
Update Auto to 1.5

### DISCUSSION THREAD

**Discussion by fishnchips2** [AbdBarho/stable-diffusion-webui-docker#558] [General]
With Automatic1111 updated to run SDXL, it would be nice to update to the latest version. I tried just changing the Auto1111 commit but there were missing resources.

**Comment by mashb1t** [comment]
tried to solve it in https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/560, runs perfectly fine for me. Let's wait for the maintainers to give feedback.

**Reply by fishnchips2** [reply]
Ah thanks, I was missing the additional import from Stability.

**Comment by AbdBarho** [comment]
https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/560 released!
